app/views/hotspots/_show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h4>Donation Sights</h4>
+<div class="row">
+    <div class="col-xs-0"></div>
+<div id = 'show_form' class="col-xs-5">
+<% @hotspots.each do |hotspot| %>
+<div class="list-group">
+  <a href="#" class="list-group-item active">
+    <h4 class="list-group-item-heading"><tbody>
+          <tr>
+           <%= hotspot.title %>
+          </tr>
+        </h4>
+    <p class="list-group-item-text"><%= hotspot.street_address %>, <%= hotspot.zip_code %></p>
+    <p class="list-group-item-text"><%= hotspot.phone_number %></p>
+    <div class="col-xs-7"></div>
+      <%= link_to 'Add', "/hotspots/new", class: 'btn-btn default' %>
```

Comment:
This and the line below this are not inside the div above it (the div is opened and closed on line 15) so this does not need to be indented. It can be even with the line above it.